tests/test_cli_flags.py

```
@@ -355,13 +411,21 @@ def test_should_use_plain_flags_override(monkeypatch):
     monkeypatch.setattr(sys.stdout, "isatty", lambda: False)
     monkeypatch.setattr(sys.stderr, "isatty", lambda: False)
-    assert _should_use_plain(plain=False, interactive=True) is False
+    use_plain, reason = _should_use_plain(plain=False, interactive=True)
+    assert use_plain is False
+    assert reason is None
 
     monkeypatch.setenv("POLYLOGUE_FORCE_PLAIN", "1")
-    assert _should_use_plain(plain=False, interactive=False) is True
+    use_plain, reason = _should_use_plain(plain=False, interactive=False)
+    assert use_plain is True
+    assert reason and "POLYLOGUE_FORCE_PLAIN" in reason
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)